newdle/api.py

```diff
@@ -38,6 +42,11 @@
     UserSearchResultSchema,
 )
 
+statesDict = {
```

Comment:
no camelcase please. `state_titles`?

```suggestion
state_titles = {
```